packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-in/java/summaryofsteps.md

```diff
@@ -0,0 +1,11 @@
+## Summary of steps
+
+This use case entails building out a simple UI to capture the username and password
```

Comment:
Nit: change to `capture the username, password and authenticating`

Reply:
Fixed in new branch vn-OKTA-408182-oie-java-embedded-app-2.